apps/search/collections/global_search_results.coffee

```diff
@@ -0,0 +1,22 @@
+_ = require 'underscore'
+Backbone = require 'backbone'
+sd = require('sharify').data
+qs = require 'querystring'
+SearchResult = require '../../../models/search_result.coffee'
+
+module.exports = class GlobalSearchResults extends Backbone.Collection
+  model: SearchResult
+
+  url: "#{sd.API_URL}/api/v1/match"
+
+  parse:  (response) ->
+    _.reject response, (item) ->
+      # HACK filter out sensitive results (at the artist's request)
+      JSON.stringify(item).match(/kippenberger|zoe.*leonard|pat.*lipsky/i)
```

Comment:
Yeah artists or their estates or representing galleries demanded complete removal, but we had little control (except via hacks like this) over Google's results. Going forward, this _may_ not be as necessary since non-public artists shouldn't appear in results any longer.